build.gradle

```diff
@@ -355,7 +366,7 @@ task cleanHailDocs(type: Exec) {
 }
 
 task setupDocsDirs(type: Exec, dependsOn: cleanHailDocs) {
-    commandLine 'mkdir', '-p', 'build/tmp/docs/', 'build/www/', 'build/tmp/python/'
+    commandLine 'mkdir', '-p', 'build/tmp/', 'build/www/', 'build/tmp/python/', 'build/tmp/docs', 'build/www/docs'
```

Comment:
AFAIK, you shouldn't need `build/tmp/` if you're using `mkdir -p build/tmp/docs`